dashboard/app/jobs/evaluate_rubric_job.rb

```diff
@@ -104,6 +138,12 @@ def perform(user_id:, requester_id:, script_level_id:, rubric_ai_evaluation_id:
     channel_id = get_channel_id(user, script_level)
     code, project_version = read_user_code(channel_id)
 
+    # Check for PII / sharing failures
+    # Get the 2-character language code from the user's preferred locale
+    locale = (user.locale || 'en')[0...2]
+    share_failure = ShareFiltering.find_share_failure(code, locale, exceptions: true)
+    raise "ShareFailure #{share_failure.type}" if share_failure
```

Comment:
This doesn't need to be here since I've added exceptions directly to `ShareFiltering`. I'll probably remove it... it would cause a general failure... but it isn't possible.

Reply:
I'll remove it along with the assignment.

Reply:
Removed. I'm using exceptions for a reason.